server/datastore/mysql/hosts.go

```diff
@@ -390,27 +390,35 @@ func (d *Datastore) DeleteHost(ctx context.Context, hid uint) error {
 	return nil
 }
 
-func (d *Datastore) Host(ctx context.Context, id uint) (*fleet.Host, error) {
-	sqlStatement := `
+func (d *Datastore) Host(ctx context.Context, id uint, skipLoadingExtras bool) (*fleet.Host, error) {
+	policiesColumns := `,
+		       coalesce(failing_policies.count, 0) as failing_policies_count,
+		       coalesce(failing_policies.count, 0) as total_issues_count`
+	policiesJoin := `
+			JOIN (
+		    	SELECT count(*) as count FROM policy_membership WHERE passes=0 AND host_id=?
+			) failing_policies`
```

Comment:
Aren't you missing the `ON (h.id=failing_policies.host_id)` clause?

Reply:
`ON` is syntactic sugar around the where, which we are passing.